packages/core/src/extensions/SlashMenu/defaultSlashMenuItems.ts

```diff
@@ -1,13 +1,34 @@
 import { BlockNoteEditor } from "../../BlockNoteEditor";
-import { BlockSchema, PartialBlock } from "../Blocks/api/blockTypes";
+import { Block, BlockSchema, PartialBlock } from "../Blocks/api/blockTypes";
 import { defaultBlockSchema } from "../Blocks/api/defaultBlocks";
 import { imageToolbarPluginKey } from "../ImageToolbar/ImageToolbarPlugin";
 import { BaseSlashMenuItem } from "./BaseSlashMenuItem";
 
+function setSelectionToNextContentEditableBlock<BSchema extends BlockSchema>(
```

Comment:
can you add comments to explain the changes to this file? Seems like some significant logic has been changed and it's unclear to me why

Reply:
Our logic when inserting/selecting an "add image button" works differently to Notion, mainly because Notion blurs the editor while we don't. So the added code here makes the text cursor go to the next block with editable content instead of selecting the button, since selecting the button doesn't open the image replacement toolbar anyway. Do you think we should change the behaviour to match Notion (probably in a different PR)?